src/olympia/lib/settings_base.py

```diff
@@ -1496,15 +1496,39 @@ def get_sentry_release():
     return version
 
 
-# This is the DSN to the Sentry service.
+def scrub_sensitive_data(event, hint):
+    def _scrub_sensitive_data_recursively(data, name=None):
+        # This only works with lists or dicts but we shouldn't need anything else.
+        if isinstance(data, (list, dict)):
+            items = data.items() if isinstance(data, dict) else enumerate(data)
+            for key, value in items:
+                data[key] = _scrub_sensitive_data_recursively(value, name=key)
+        elif isinstance(data, str) and name in SENTRY_SENSITIVE_FIELDS:
+            data = '*** redacted ***'
+        return data
+
+    try:
+        event = _scrub_sensitive_data_recursively(event)
+    except Exception:
+        pass
+    return event
```

Comment:
I know this isn't a new thing, but I don't like how we have random functions in the middle of the settings file.  Perhaps they should be at the top at least?

Reply:
I agree in theory, but it references `SENTRY_SENSITIVE_FIELDS` which is defined below with the rest of the sentry stuff. So I'm not sure moving it to the top is better...